integreat_cms/cms/signals/FeedbackUpdateHandler.py

```diff
@@ -0,0 +1,60 @@
+from django.db.models.signals import post_delete, post_save
```

Comment:
Python usually uses `snake_case` for file naming, so I suggest to call this file `feedback_signals.py` or similar.

Reply:
Thank you for your review, I have renamed now the file as you suggested :+1: